packages/mongodb-chatbot-server/src/routes/conversations/addMessageToConversation.test.ts

```diff
@@ -128,6 +129,39 @@ describe("POST /conversations/:conversationId/messages", () => {
       origin,
     });
   });
+  test("uses previous message filter", async () => {
+    const mockMsg = {
+      role: "user",
+      // This corresponds to ~130 tokens
+      content:
+        "In a quaint village nestled among verdant hills, a whimsical inventor, Eleanor, toiled in her sun-dappled workshop. Every day, she crafted extraordinary gadgets, each more intricate than the last. One morning, as the village awoke to a chorus of songbirds, Eleanor unveiled her latest creation: a clockwork bird with iridescent feathers. This mechanical marvel could mimic any tune it heard. The villagers gathered, marveling as the bird sang melodies from distant lands. Meanwhile, beyond the rolling meadows, a curious fox watched, its amber eyes reflecting a world where nature and Eleanor's inventions coexisted in harmonious wonder.",
```

Comment:
Eerily close to this AI generated story! https://storybird.ai/library/children-stories/the-mystery-of-good-view-village

Reply:
we were probably using similar ChatGPT prompts lol.

i hadn't seen storybird.ai before. that's neat. very very similar to [FableFusion](https://github.com/bpmutter/fablefusion), my first gen AI app. i wasn't able to figure out the image generation part (this was back in ~dec '22). curious how they handled it.